controllers/nnf_workflow_controller.go

```diff
@@ -82,12 +82,12 @@ type NnfWorkflowReconciler struct {
 //+kubebuilder:rbac:groups=dws.cray.hpe.com,resources=computes,verbs=get;create;list;watch;update;patch
 //+kubebuilder:rbac:groups=cray.hpe.com,resources=lustrefilesystems,verbs=get;list;watch
 
+//+kubebuilder:rbac:groups=nnf.cray.hpe.com,resources=nnfcontainerprofiles,verbs=get;create;list;watch;update;patch;delete;deletecollection
+//+kubebuilder:rbac:groups=batch,resources=jobs,verbs=get;list;watch;create;update;patch;delete;deletecollection
```

Comment:
You don't have a watch on jobs so the controller won't receive Reconcile()s. Is this OK because you are polling?

Reply:
Then do you need the watch? I guess other resources aren't all that tidy either. So maybe leave it.

Reply:
Doesn't seem like we do. I'll remove it.